mongo-fixes/neptune-migration/neptune-migration.sh

```diff
@@ -0,0 +1,32 @@
+
+if [ -z "$1" ]
```

Comment:
use of double brackets is more current
```suggestion
if [[ -z "$1" ]]
```